source/features/easy-toggle-files.tsx

```diff
@@ -3,13 +3,25 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-function toggleFile(event: Event): void {
+const toggleFileButtonSelector = '[aria-label="Toggle diff contents"]';
+
+function toggleFile(event: MouseEvent): void {
 	const elementClicked = event.target as HTMLElement;
-	const headerBar = elementClicked.closest<HTMLElement>('.file-header.file-header--expandable')!;
+	const headerBar = elementClicked.closest<HTMLElement>('.file-header')!;
 
 	// The clicked element is either the bar itself or one of its 2 children
 	if (elementClicked === headerBar || elementClicked.parentElement === headerBar) {
-		select('[aria-label="Toggle diff contents"]', headerBar)!.click();
+		if (event.altKey) {
+			const toggleFileButtonState = select(toggleFileButtonSelector, headerBar)!.getAttribute('aria-expanded');
+
+			for (const button of select.all(toggleFileButtonSelector)) {
+				if (button.getAttribute('aria-expanded') === toggleFileButtonState) {
+					button.click();
+				}
+			}
+		} else {
+			select(toggleFileButtonSelector, headerBar)!.click();
+		}
```

Comment:
Can you match the pattern used in https://github.com/sindresorhus/refined-github/blob/master/source/features/toggle-everything-with-alt.tsx? Perhaps you can `export clickAll` in it

Reply:
Wait, that shouldn’t be necessary actually. The toggle button already has that listener. All this function I should do is passing the click event from the header to the toggle button. 

I think you can drop most of the new code here and just use .dispatchEvent instead of .click. I don’t know if you can reuse the user’s event or if you need to generate a new MouseEvent